pkg/agent/controller/traceflow/packetin.go

```diff
@@ -34,11 +34,16 @@ import (
 	binding "antrea.io/antrea/pkg/ovs/openflow"
 )
 
+var skipTraceflowUpdateErr = errors.New("Skip Traceflow Update")
```

Comment:
nit: `errors.New("skip Traceflow update")`  to follow the convention that error shouldn't be capitalized first letter unless proper noun, even though it's not appended to any log currently?